pkg/virt-handler/device-manager/device_plugin_base_test.go

```diff
@@ -79,4 +82,79 @@ var _ = Describe("Device plugin base", func() {
 			Expect(fakeServer.count).To(Equal(3))
 		})
 	})
+
+	Context("device plugin lifecycle", func() {
+		var workDir string
```

Comment:
You can just use `DeferCleanup` to avoid sharing this variable as well for the `stop`